lib/mail/field.rb

```diff
@@ -232,12 +232,13 @@ def split(raw_field)
     #  that is immediately followed by WSP.  Each header field should be
     #  treated in its unfolded form for further syntactic and semantic
     #  evaluation.
-    def unfold(string)
-      string.gsub(/[\r\n \t]+/m, ' ')
+    def unfold(name, value)
+      regex = name == 'Subject' ? /\s*\r\n\s+/m : /[\r\n \t]+/m
```

Comment:
This feels a little inelegant to me. Is there anyway to just modify the original regex?